hyperspy/_signals/eds_tem.py

```diff
@@ -641,3 +641,61 @@ def _get_dose(self, method, beam_current='auto', real_time='auto',
             return real_time * beam_current * 1e-9 / constants.e
         else:
             raise Exception('Method need to be \'zeta\' or \'cross_section\'.')
+
+def cross_section_to_zeta(cross_sections, elements):
```

Comment:
What about a more explicit name for the function e.g. `edx_cross_sections_to_zeta_factors`?